renderer/reducers/modal.js

```diff
@@ -39,6 +42,32 @@ const createModalData = (type, options) => ({
 // Actions
 // ------------------------------------
 
+/**
+ * openDialog - Open dialog.
+ *
+ * @param  {string} id Name of modal to open
+ * @returns {Function} Thunk
+ */
+export const openDialog = id => dispatch => {
```

Comment:
What do you think about moving these dialog methods to a new `dialog` reducer? (I know that these particular dialogs are technically modals too, but they don't live in the modal stack and aren't related to the non-dialog modals)